core/src/main/scala/org/bitcoins/core/wallet/fee/FeeUnit.scala

```diff
@@ -9,8 +9,14 @@ import org.bitcoins.core.protocol.transaction.Transaction
   */
 sealed abstract class FeeUnit {
   def currencyUnit: CurrencyUnit
+  def *(cu: CurrencyUnit): CurrencyUnit = this * cu.satoshis.toLong
+  def *(int: Int): CurrencyUnit = this * int.toLong
```

Comment:
What unit is `int` here? Aren't you just moving the original problem you had into this? 

Same with `def *(long: Long)`